package.json

```diff
@@ -5,24 +5,28 @@
   "main": "dist/index.js",
   "scripts": {
     "clean": "rimraf dist",
-    "transpile": "tsc",
+    "build": "npm run clean && npm run make-dist && npm run transpile",
```

Comment:
I've separated the build into two steps.  The first step (prepare) runs when the user runs `npm install` and will create the generated protobuf files but it will put them into `src`.